azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -178,10 +194,8 @@ private void cancelFlowQuietly(ExecutableFlow flow, Status originalStatus) {
           "Cleaning up stale flow " + flow.getExecutionId() + " in state " + originalStatus
               .name());
       this.containerizedDispatchManager.cancelFlow(flow, flow.getSubmitUser());
-    } catch (ExecutorManagerException eme) {
-      logger.error("ExecutorManagerException while cancelling flow.", eme);
-    } catch (RuntimeException re) {
-      logger.error("Unexpected RuntimeException while finalizing flow during clean up." + re);
+    } catch (Exception e) {
```

Comment:
If the idea is to catch everything, catch "Throwable" instead of Exception.

Reply:
It seems Exception should be sufficient. The cleanup action is schedule in a frequent manner. A failure does not affect the performance.